src/Enums/CsvUserPermissionsEnum.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace EscolaLms\CsvUsers\Enums;
+
+use EscolaLms\Core\Enums\BasicEnum;
+
+class CsvUserPermissionsEnum extends BasicEnum
+{
+    const CSV_USERS_EXPORT = 'csv users export';
```

Comment:
@mcholcha please let us know, what the new patter for permission names

Reply:
https://github.com/EscolaLMS/GH-Actions-Template/blob/main/readme.md